resources/harpy.yaml

```
@@ -12,4 +13,5 @@ dependencies:
   - snakemake-minimal >7
   - samtools =1.20
   - seqtk
-  - tabix
\ No newline at end of file
+  - tabix
+  - gocryptfs
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)